### PR TITLE
Update pulp packages that need python-jsonschema updates

### PR DESCRIPTION
**Pull request by Odilhao**

### CODE REVIEW

**Review by pcreech — APPROVED** [review]

File: packages/python-pulp-rpm-client/python-pulp-rpm-client.spec [hunk 1]
```diff
@@ -60,6 +60,8 @@ set -ex
 
 %files -n %{?scl_prefix}python%{python3_pkgversion}-%{pypi_name}
 %doc README.md
+%exclude %{python3_sitelib}/pulpcore/__pycache__/*
+%exclude %{python3_sitelib}/pulpcore/__init__.py
```
Comment (pcreech) [inline packages/python-pulp-rpm-client/python-pulp-rpm-client.spec:64]: Is there a specific reason we are excluding ```__init__.py```?

Comment (Odilhao) [reply packages/python-pulp-rpm-client/python-pulp-rpm-client.spec:64]:
```
Error: Transaction test error:
          file /usr/lib/python3.9/site-packages/pulpcore/__init__.py from install of python39-pulp-rpm-client-3.18.5-1.el8.noarch conflicts with file from package python39-pulpcore-3.21.0-1.el8.noarch
          file /usr/lib/python3.9/site-packages/pulpcore/__pycache__/__init__.cpython-39.opt-1.pyc from install of python39-pulp-rpm-client-3.18.5-1.el8.noarch conflicts with file from package python39-pulpcore-3.21.0-1.el8.noarch
          file /usr/lib/python3.9/site-packages/pulpcore/__pycache__/__init__.cpython-39.pyc from install of python39-pulp-rpm-client-3.18.5-1.el8.noarch conflicts with file from package python39-pulpcore-3.21.0-1.el8.noarch

```

Comment (Odilhao) [reply packages/python-pulp-rpm-client/python-pulp-rpm-client.spec:64]: It conflicts with the `__init__.py` from pulpcore

Comment (pcreech) [reply packages/python-pulp-rpm-client/python-pulp-rpm-client.spec:64]: looks like it conflicts with pulpcore-client provided __init__.py (makes sense)

**Review by pcreech — APPROVED** [review]

**Pull request merged**